.github/workflows/nextjs-bundle-analysis-annotation.yml

```diff
@@ -57,22 +57,22 @@ jobs:
           echo ::set-output name=body::$body
 
       - name: Find Comment
-        uses: peter-evans/find-comment@v2
+        uses: peter-evans/find-comment@v3.1.0
```

Comment:
I’m pretty sure the newer versions of this have data encoding issues. Let’s look and find an old PR I worked on to verify.

Reply:
Hmm, also found issue in the latest version of /create-or-update-comment. Let's keep it as it is for now